### DISCUSSION TITLE
Question about FSK settings for speed

### DISCUSSION THREAD

**Discussion by tauesdospi** [jgromes/RadioLib#930] [General]
Hi,
I'm using SX1262 in ESP32 and I'm looking for the best combiation of parameters for the maximum performance (bitrate)
The question is about the relations between datarate <-> frequency deviation <-> rx bandwidth. At this point I'm not considering other parameters like output power, antennas gain, etc.. I'm considering these constants and I only want to tune the first ones.
Anybody know any document / formula / etc which may assist on determining the best combination for  such parameters?

Thanks
Antonio

**Comment by jgromes** [comment]
I think this is a pretty generic RF question, not sure if this is the best place to ask it. In general if you need high bit rate then you should probably use higher frequency deviation (and matching Rx bandwidth) to keep reasonable modulation index. But again, better to ask this in some RF engineering forum.

**Reply by tauesdospi** [reply]
I have figured some basic rules about this... Do you know any forum or place I can address?
Thanks!
Antonio

**Reply by jgromes** [reply]
I'm guessing [electrical engineering stack exchange](https://electronics.stackexchange.com/) could be one.
